osc-collab.py

```diff
@@ -4062,3 +4063,14 @@ def do_collab(self, subcmd, opts, *args):
 
     else:
         raise RuntimeError('Unknown command: %s' % cmd)
+
+def identify_conf():
```

Comment:
What about `self.options.conffile`?  that's no longer checked here.

Reply:
I've added a commit for this.